app/views/travel_insurance_registrations/medical_conditions_questionaire_form.html.erb

```diff
@@ -0,0 +1,73 @@
+<%= heading_tag(t(registration_title), level: 1) %>
+<p><%= t('travel_insurance_registrations.intro') %></p>
+
+<h3>Step 4 of 4 </h3>
+
+<div class="l-2col-even-row">
+  <div class="l-2col">
+    <section class="l-2col-main" data-wizard-form>
+      <%= form_for @form, url: medical_conditions_questionaire_travel_insurance_registrations_path, html: { class: 'form' }, builder: Dough::Forms::Builders::Validation do |f| %>
```

Comment:
We'd prefer to use a more descriptive classname here than `form`. If we are applying styles to forms in general we can use the element name but we prefer to be more specific than that.